docs/alerts/monitors/create-monitor.md

```diff
@@ -108,242 +116,244 @@ For guidance on optimizing scan costs when using Flex Pricing, refer to the [FAQ
+* A **returned row count** (default), which is the number of rows returned from the log search.<br/><img src={useBaseUrl('img/alerts/monitors/trigger-alerts-field.png')} alt="trigger alerts on field" style={{border: '1px solid gray'}} width="200"/>
 * A numeric field returned from the search. You can pick any numeric field from your query, and alert on the value of that field. The field is `_count` in the above screenshot. To convert a string to a number use the [`num` operator](/docs/search/search-query-language/search-operators/num). For example, if you have a field named **duration**, you would use the `num` operator as follows to convert it to a number value.
    ```sh
    | num(duration)
    ```
 
+Triggers are evaluated by balancing the requirement of timely alert notifications while ensuring that monitor data is indeed available to evaluate trigger conditions.
+
+* For [static logs monitors](#static-detection-method), triggers are similar to "Alert when the result is greater than _ within Y Minutes". The triggers are evaluated periodically as below.
+   | When detection window (Y) is | Evaluate trigger every |
+   |:-----------------------------|:-----------------------|
+   | 30m or less  | 1m  |
+   | 30m to 3h    | 2m |
+   | 3hr to 12h   | 10m  |
+   | Greater than 12h  | 20m |
+* For [anomaly logs monitors](#anomaly-detection-method), triggers are evaluated every `timeslice` as specified in the monitor query. For example, the below query is evaluated every 2 minutes.
+   ```
+   _sourceCategory=Labs/Apache/Access
+   | timeslice 2m
+   | parse "HTTP/1.1\" * " as status_code
+   | if (status_code = "200", 1, 0) as successes
+   | if (status_code = "404", 1, 0) as fails
+   | sum(successes) as success_cnt, sum(fails) as fail_cnt by _timeslice
+   | (fail_cnt/(success_cnt+fail_cnt)) * 100 as failure_rate_pct
+   ```
+* For [outlier logs monitors](#outlier-detection-method), triggers are evaluated every 5 minutes.
+
+When configuring monitor trigger conditions, you can set a resolution window to resolve alerts quickly once the underlying issue is fixed. The resolution window specifies how long a monitor will wait before resolving an alert after the issue is corrected.
+
+For example, if your monitor evaluates the last 1 hour, you can set a resolution window of 15 minutes. Once the resolution window is continuously satisfied for 15 minutes, the alert will resolve automatically.<br/><img src={useBaseUrl('img/alerts/monitors/config-resolution-window-2.png')} alt="config-resolution-window" style={{border: '1px solid gray'}} width="700"/>
+
 #### Static detection method
 
-**Example: Log Trigger Type - Critical and Warning**  
+**Example: Logs - Static - Critical and Warning**  
 
 <img src={useBaseUrl('img/alerts/monitors/logs-trigger-type.png')} alt="logs trigger type.png" style={{border: '1px solid gray'}} width="600"/>
 
 `Alert when returned row count is <threshold type> <threshold> within <time range>`
```

Comment:
To match the screenshot, should this be `Alert when result is <threshold type> <threshold> within <time range>`?